guides/common/modules/proc_upgrading-project-or-proxy-in-place-using-leapp.adoc

```diff
@@ -13,12 +13,29 @@ ifdef::satellite[]
 * Review Known Issues before you begin an upgrade.
 For more information, see {ReleaseNotesURL}ref_known-issues_assembly_introducing-red-hat-satellite[Known Issues in {ProjectName} {ProjectVersion}].
 endif::[]
-* Access to available repositories or a local mirror of repositories.
 * If you previously upgraded {Project} or {SmartProxy} from an earlier version, and the `/var/lib/pgsql` contained the PostgreSQL database content before the migration from PostgreSQL 9 to PostgreSQL 12 from the SCL, empty `/var/lib/pgsql` before proceeding.
 * During the upgrade, the PostgreSQL data is moved from `/var/opt/rh/rh-postgresql12/lib/pgsql/data/` to `/var/lib/pgsql/data/`.
 If these two paths reside on the same partition, no further action is required.
 If they reside on different partitions, ensure that there is enough space for the data to be copied over.
 You can move the PostgreSQL data on your own and the upgrade will skip this step if `/var/opt/rh/rh-postgresql12/lib/pgsql/data/` does not exist.
+
+ifdef::satellite[]
+.Prerequisites for Disconnected Environment
+If you run {Project} in a disconnected environment, ensure it meets the following prerequisites:
+
+* You must obtain and deploy Leapp metadata manually.
+For more information, see https://access.redhat.com/articles/3664871[Leapp utility metadata in-place upgrades of RHEL for disconnected upgrades].
+* You require access to {RHEL} and {Project} packages.
+* Obtain the ISO files for {RHEL} 8 and {Project}.
```

Comment:
```suggestion
* You require access to {RHEL} and {Project} packages.
Obtain the ISO files for {RHEL} 8 and {Project}.
```
This is a bit confusing. Are the ISOs the way how I get access to the packages? If it is so, I would make it one bullet point.